netlify.toml

```diff
@@ -2,5 +2,10 @@
   command = "npm run build"
   publish = "dist"
 
+# Change the version of Hugo you want to use on Netlify
+# by changing this environment variable.
+[build.environment]
+  HUGO_VERSION = "0.21"
```

Comment:
Hugo oftentimes uses SemVer version strings when defining the version used, especially during active development, e.g. `v0.21-DEV`. Is it possible to set `build.environment` using globs or wildcards to prevent lock-in on an older version of Hugo?